modules/core/shared/src/main/scala/Session.scala

```diff
@@ -365,7 +365,7 @@ object Session {
      * isn't running arbitrary statements then `minimal` might be more efficient.
      */
     def full[F[_]: Monad]: Recycler[F, Session[F]] =
-      closeEvictedPreparedStatements[F] <+> ensureIdle[F] <+> unlistenAll <+> resetAll
```

Comment:
No longer needed since we now close any evicted statement as soon as possible after cache mutation (either on an insert or get).